bin/osg-topology

```diff
@@ -168,7 +160,9 @@ def main():
     oparser = get_parser()
 
     args = oparser.parse_args()
-    if getattr(args, "filter") is not None:
+    if not args.which:
+        oparser.error(f"No command specified; see {sys.argv[0]} --help for usage")
+    if getattr(args, "filter", None) is not None:
```

Comment:
Is the `None` default necessary?  I am wondering if `oparser.parse_args()` won't automatically set the `filter` attribute to `None` by default.

Not that it hurts anything to provide the default, though.

Reply:
It will not. I think I can add `default=None` to the `add_argument()` calls above to get the same behavior though.